tests/vagrant/templates/Vagrantfile.tmpl

```diff
@@ -0,0 +1,40 @@
+# -*- mode: ruby -*-
```

Comment:
tests/vagrant/templates/Vagrantfile.tmpl should be removed, as it's not used anymore (Vagrantfiles are generated automatically by `aqrdev` now)